docs/src/components/document-renderer.tsx

```diff
@@ -97,13 +99,26 @@ const getRenderers = (
         />
       );
     },
-    image: ({ src, alt }) => (
-      <img
-        className="rounded-lg my-2"
-        src={`/images/content/${slug}/${src}`}
-        alt={alt}
-      />
-    ),
+    image: ({ src, alt, title }) => {
```

Comment:
WRT my other comment of removing images from the document editor, perhaps we can add "Caption" to the cloud image?

Reply:
Great point!